pkg/config/config.go

```diff
@@ -199,7 +200,8 @@ func init() {
 	Datadog.SetDefault("kubelet_client_key", "")
 
 	Datadog.SetDefault("kubernetes_collect_metadata_tags", true)
-	Datadog.SetDefault("kubernetes_metadata_tag_update_freq", 60*5) // 5 min
+	Datadog.SetDefault("kubernetes_metadata_tag_update_freq", 60*5)    // 5 min
+	BindEnvAndSetDefault("kubernetes_map_services_on_reference", true) // temporary opt-out of the new mapping logic
```

Comment:
This comment is somewhat confusing, I thought we were opting out of the logic we introduce here, whereas it's actually the opposite (we disable the old one). Can we clarify?